deisctl/README.md

```diff
@@ -84,7 +84,7 @@ Done.
 
 Note that the default start command activates 1 of each component.
 You can scale components with `deisctl scale router=3`, for example.
-The router and the registry are the only component that _currently_ scales beyond 1 unit.
+The router, the registry and the store gateway are the only component that _currently_ scales beyond 1 unit.
```

Comment:
Note for @carmstrong: subject-verb disagreement here should be cleaned up when you submit your other PR.